src/atoms/ArrowBullet.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styled from 'styled-components';
+
+import { FontIcon, getColor } from '..';
+
+const Bullet = styled.a`
+    color: ${getColor('type')};
+    font-size: ${props => props.theme.variables.headingMediumSize};
```

Comment:
There is a getVariable() alongside getColor(), if you want to rid some of this syntax.

Reply:
Done, thanks